buddyboss/buddyboss-platform
# #1645 - Fixed Custom Notification returns blank text in notification …

…issue

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1645  .

### How to test the changes in this Pull Request:


1. Add this code snippet to your theme or plugin https://www.codepile.net/pile/nNz16GvR
2. Refresh page
3. Check notification popup & notification page > https://prnt.sc/v0pyx9

### Proof Screenshots or Video


![image](https://user-images.githubusercontent.com/65785731/96429649-d06c2800-121e-11eb-97cd-f6476a91803a.png)


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
